Source/Analytics/Read/HealthRisks/HealthRiskName.cs

```diff
@@ -0,0 +1,12 @@
+using Concepts.HealthRisks;
```

Comment:
Should have a header:
```suggestion
/*---------------------------------------------------------------------------------------------
*  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
*  Licensed under the MIT License. See LICENSE in the project root for license information.
*--------------------------------------------------------------------------------------------*/

using Concepts.HealthRisks;
```